roscala/src/main/scala/coop/rchain/roscala/ob/Ctxt.scala

```diff
@@ -22,20 +22,27 @@ class Ctxt(var tag: Location,
            var monitor: Monitor)
     extends Ob {
 
+  private val _rslt = new AtomicReference[Ob](Niv)
```

Comment:
Can you add some comments here why we are doing that?
Also I fear that we'll have to do the same for `argvec` and `env` (see `doRtn` and `Location.store`) since a `Ctxt` can write into its parent's `argvec` as well.

Reply:
I think you're right. I've been thinking about a better solution for atomic fields, maybe a special macro.